src/app/pages/courses/courses.template.html

```diff
@@ -1,8 +1,11 @@
 <div class="home-page">
 	<toolbox></toolbox>
 	<h4>Courses list:</h4>
-	<div class="todo-list">
-		<course *ngFor="let course of courseList" [course]="course" (deleteCourseEvent)="deleteCourseFromCoursesList($event)"></course>
+	<div *ngIf="!courseList.length">
+		<p>No data. Feel free to add new course.</p>
+	</div>
+	<div class="todo-list" *ngIf="courseList.length">
+		<course *ngFor="let course of courseList | orderBy" [course]="course" (deleteCourseEvent)="deleteCourseFromCoursesList($event)"></course>
```

Comment:
It would me much easier to read if you put attributes in one column
```html
<course 
    *ngFor="let course of courseList | orderBy"
    [course]="course" 
    (deleteCourseEvent)="deleteCourseFromCoursesList($event)"></course>
```
Or
```html
<course 
    *ngFor="let course of courseList | orderBy"
    [course]="course" 
    (deleteCourseEvent)="deleteCourseFromCoursesList($event)"
></course>
```